canmatrix/canmatrix.py

```diff
@@ -428,9 +428,9 @@ def setMax(self, max=None):
         return self.max
             
     def calcMax(self):
-        rawMax = self.calculateRawRange()[1]
+        rawMax = Decimal(self.calculateRawRange()[1])
 
-        return self.offset + (rawMax * self.factor)
+        return self._offset + (Decimal(rawMax) * Decimal(self._factor))
```

Comment:
I think that specifying the `float_factory` would be sufficient and `calcMax()` wouldn't need to change.  But, I'm also guessing that `float_factory` isn't presently accessible from the CLI interface.  Maybe we could just make `decimal.Decimal` default?  It'd be a breaking change, but maybe for the better.  I suppose regardless we'd want control from the CLI.